benchmark.js

```diff
@@ -787,12 +787,11 @@
      * @static
      * @memberOf Benchmark
      * @param {number} number The number to convert.
+     * @param {locale} string Identifying string of the locale to use.
      * @returns {string} The more readable string representation.
      */
-    function formatNumber(number) {
-      number = String(number).split('.');
-      return number[0].replace(/(?=(?:\d{3})+$)(?!\b)/g, ',') +
-        (number[1] ? '.' + number[1] : '');
+    function formatNumber(number, locale) {
+      return number.toLocaleString(locale);
     }
```

Comment:
Could you add a check during the creation of `formatNumber` that `Number.prototype.toLocaleString` is a function and if it's not then make `formatNumber` fallback to the legacy formatter?

Reply:
Is like this ok or did you mean something else with _during the creation of formatNumber_?

Reply:
That'll do!